.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "yaml.schemas": {
```

Comment:
do we want to include this file?

Reply:
I was unsure about this but looking at it now it isnt up to date. I'm going to take it out. Thanks @mocsharp